modules/osdk-run-proxy.adoc

```diff
@@ -0,0 +1,142 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/golang/osdk-golang-tutorial.adoc
+// * operators/operator_sdk/ansible/osdk-ansible-tutorial.adoc
+// * operators/operator_sdk/helm/osdk-helm-tutorial.adoc
+
+ifeval::["{context}" == "osdk-golang-tutorial"]
+:golang:
+:app-proper: Memcached
+:app: memcached
+:group: cache
+endif::[]
+ifeval::["{context}" == "osdk-ansible-tutorial"]
+:ansible:
+:app-proper: Memcached
+:app: memcached
+:group: cache
+endif::[]
+ifeval::["{context}" == "osdk-helm-tutorial"]
+:helm:
+:app-proper: Nginx
+:app: nginx
+:group: demo
+endif::[]
+
+[id="osdk-run-proxy_{context}"]
+
+= Enabling proxy support
+
+To support proxied clusters, your Operator must inspect the environment for the following standard proxy variables and pass the values to Operands:
+
+- `HTTPS_PROXY`
+- `HTTPS_PROXY`
+- `NO_PROXY`
+
+.Procedure
+ifdef::golang[]
+. Add the `proxy.ReadProxyVarsFromEnv` helper function to the reconcile loop in the `controllers/memcached_controller.go` file and append the results to the Operand environments:
++
+[source,golang,subs="attributes+"]
```

Comment:
This `subs="attributes+"` isn't necessary unless you are actually trying to insert attributes that you want to be replaced

(Same comment for all below code blocks that use it too)